.github/workflows/staging-scheduled-deploy.yml

```diff
@@ -3,8 +3,8 @@ name: Scheduled Deploy From Main to Staging
 on:
   # schedule:
   #   - cron: '0 10 * * 1-5'
-  # Disable Schedule for Pen Testing
```

Comment:
Did you mean to remove this? It seems like a relevant explanation to leave in place until pen-testing is done.

Reply:
good catch, added back